### ISSUE TITLE
Reset center of rotation after transforms

### ISSUE THREAD

**Issue by yijiang1** [OpenChemistry/tomviz#589]
The default rotation center is at the center of the matrix:
<img width="318" alt="screen shot 2016-09-05 at 12 54 24 pm" src="https://cloud.githubusercontent.com/assets/13088588/18254092/623c39be-7368-11e6-817b-95f40fb218ff.png">

If I do some transform that changes data size, say downsample by 2, then the rotation center often ends up at the corner like this
<img width="447" alt="screen shot 2016-09-05 at 12 54 36 pm" src="https://cloud.githubusercontent.com/assets/13088588/18254104/8ad671be-7368-11e6-90db-c8496d85013c.png">

Is that possible to reset the rotation center to the center of the matrix after transforms?

**Comment by cquammen** [comment]
Yes in theory, but in some cases that could lead to annoying behavior. What if you have two images open that occupy different spaces and you want to apply some transforms to both? Then the center will move to whichever transform was performed last, which could be just as irritating as the behavior now. Moreover, if the transforms for both images re-execute, the center could jump around as you are interacting with the visualization. There could even be a race condition between the thread where the transform is running and the main UI race condition.

**Comment by yijiang1** [comment]
I can't think of any situations when people want the rotation center to be somewhere else other than the center of the matrix. I'd even automatically reset the rotation center if a transform changes matrix size.
In general, I think the best way to handle multiple datasets is to put them in separate RenderView windows.

**Comment by Hovden** [comment]
Can we just center all datasets to the origin? This is a recurring difficulty for users.
